### ISSUE TITLE
[Keyboard Navigation – Make Code - Home page] Keyboard focus indicator is not visible on multiple controls under ‘Home page’. 

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3510]
**User Experience**:
If the desired component, on which keyboard operations will interact at any point in time is not visually determined, people with attention limitations, short term memory limitations, or limitations in executive processes will find it difficult to discover where the focus is located.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that keyboard focus indicator is visible on (Micro:bit, More, Microsoft logo, Import button) controls or not.

**Actual Result:**
Keyboard focus indicator is not visible on (Micro:bit, More, Microsoft logo, Import button) controls on ‘Home Page’ and due to this, visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Expected Result:**
Keyboard focus indicator should be visible on (Micro:bit, More, Microsoft logo, Import button) controls on ‘Home Page’, so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG 2.4.7 - Focus Visible
](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#A11yMAS, #Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.7, #Benchmark, #Win10-Edge(Chromium)

**Comment by Shalini3457** [comment]
![WCAG2 4 7_Focus indicator is not visible](https://user-images.githubusercontent.com/72871089/97535206-4b80cb80-19e1-11eb-8626-d6d79bf3c50d.jpg)

**Comment by abchatra** [comment]
This is visible in high contrast mode. Please validate again.
